File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/composite/transport/Car.java
// Car.java

package com.bigboxcode.designpattern.composite.transport;

public class Car implements Transport {
    @Override
    public void start() {
        System.out.println("Starting Car...");
    }

    @Override
    public void operate() {
        System.out.println("Driving Car");
    }

    @Override
    public void stop() {
        System.out.println("Stopping Car...");
    }
}

File: design-patterns/ts/composite/transport-list/car.ts
// car.ts

import Transport from "./transport";


class Car implements Transport {
    start(): void {
        console.log("Starting Car...");
    }

    operate(): void {
        console.log("Driving Car");
    }

    stop(): void {
        console.log("Stopping Car...");
    }
}

export default Car;